admin/class-boldgrid-backup-admin-premium-features.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * File: class-boldgrid-backup-admin-premium-features.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      1.12.4
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/admin
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+use Boldgrid\Backup\Admin\Card;
+
+/**
+ * Class: Boldgrid_Backup_Admin_Premium_Features
+ *
+ * @since 1.12.4
+ */
+class Boldgrid_Backup_Admin_Premium_Features {
+
```

Comment:
Generally you don't want 2 blank lines. phpcs should report this.

Reply:
Understood